test/lightning/workflows/edge_test.exs

```diff
@@ -5,10 +5,27 @@ defmodule Lightning.Workflows.EdgeTest do
 
   describe "changeset/2" do
     test "valid changeset" do
-      changeset = Edge.changeset(%Edge{}, %{workflow_id: Ecto.UUID.generate()})
+      changeset =
+        Edge.changeset(%Edge{}, %{
+          workflow_id: Ecto.UUID.generate(),
+          condition: :on_job_success
+        })
+
       assert changeset.valid?
     end
 
+    test "edges must have a condition" do
+      changeset =
+        Edge.changeset(%Edge{}, %{workflow_id: Ecto.UUID.generate()})
+        |> IO.inspect()
```

Comment:
Do we need this inspect?

Reply:
ack. nope. forgot to remove it, thank you!